orc8r/cloud/go/obsidian/swagger/remote_spec.go

```diff
@@ -24,17 +24,17 @@
 	"github.com/golang/glog"
 )
 
-type remoteSpec struct {
-	// service name of the remoteSpec
+type RemoteSpec struct {
+	// service name of the RemoteSpec
 	// should always be lowercase to match service registry convention
 	service string
 }
 
-func NewRemoteSpec(serviceName string) remoteSpec {
-	return remoteSpec{service: strings.ToLower(serviceName)}
+func NewRemoteSpec(serviceName string) RemoteSpec {
```

Comment:
P.S. add doc comments for the public objects/methods in this file

Reply:
Sounds good, I was thrown off about whether to include it since the interface of mconfig builders didn't comment its public function.